### PR TITLE
Seo 181194 intall docs courage issues hotfix 25.1.35

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Hi @gowthamanthavasiyappan 
Please review changes made
Regards,
Faith

### CODE REVIEW

File: Common/Essential-Studio/Release-notes/v25.1.38.md [hunk 1]
```diff
@@ -111,39 +111,39 @@ Please refer to this <a href="/common/essential-studio/licensing/license-key">he
...
+            {{ '[https://help.syncfusion.com/wpf/release-notes/v25.1.38?type=all](https://help.syncfusion.com/wpf/release-notes/v25.1.38)' | markdownify }}
         </td>
     </tr>
     <tr>
         <td>
             UWP
         </td>
         <td>
-            {{ '[https://help.syncfusion.com/uwp/release-notes/v25.1.38](https://help.syncfusion.com/uwp/release-notes/v25.1.38)' | markdownify }}
+            {{ '[https://help.syncfusion.com/wpf/release-notes/v25.1.38?type=all](https://help.syncfusion.com/wpf/release-notes/v25.1.38?type=all)' | markdownify }}
```
Comment (gowthamanthavasiyappan) [inline Common/Essential-Studio/Release-notes/v25.1.38.md:130]: Why you have changed the platform name?

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine

**Pull request merged**